app/Entry.js

```diff
@@ -42,15 +42,14 @@ class Entry extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      initialRouteName: 'true',
+      initialRouteName: null,
     };
   }
 
   async componentDidMount() {
```

Comment:
Delete the async if you are no using await
```suggestion
   componentDidMount() {
```